index.html

```diff
@@ -146,58 +166,71 @@ <h1 class="about">About me</h1>
+                <li class="headli">
+                    <div><img src="Images/Rectangle56.png" class="rectangle56" alt="shape">
+                        <p class="alignim">Frameworks</p>
+                    </div>
+                    <ul class="main-ul frame1">
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </li>
+            </ul> 
+            <ul class="main-ul skills">   
+                <li class="headli">
+                    <div><img src="Images/Rectangle57.png" class="rectangle57" alt="shape">
+                        <p class="alignim">Skills</p>
+                    </div>
+                    <ul class="main-ul skills2">
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </li>
+            </ul>
+        </div>
+        
     </section>
 
     <footer>
-        <p class="footer-p">I'm always <br>interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-        <form action="https://formspree.io/f/mdobgell" method="post">
-            <ul class="input-container">
-                <li><input type="text" name="name" id="name" placeholder="Full name" required maxlength="30"></li>
-                <li><input type="email" name="email" id="email" required placeholder="Email address"></li>
-                <li><textarea name="message" id="messagebox" cols="10" rows="10" placeholder="Enter text here" required maxlength="500"></textarea></li>
-                <li><button type="submit" class="touchbtn">Get in touch</button></li>
-            </ul>
-        </form>
+        <div class="footer-div">
+            <p class="footer-p">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        </div>
+        <div class="footer-div">
+            <form action="https://formspree.io/f/mdobgell" method="post">
+                <ul class="input-container">
+                    <li><input type="text" name="name" id="name" placeholder="Full name" required maxlength="30"></li>
+                    <li><input type="email" name="email" id="email" required placeholder="Email address"></li>
+                    <li><textarea name="message" id="messagebox" cols="10" rows="10" placeholder="Enter text here" required maxlength="500"></textarea></li>
+                    <li><button type="submit" class="touchbtn">Get in touch</button></li>
+                </ul>
+            </form>
+        </div>
+        
+       
         <ul class="footer-icons">
             <li><i class="fab fa-github"></i></li>
             <li><i class="fab fa-linkedin-in"></i></li>
```

Comment:
[OPTIONAL] Please consider centring your `contact form` and `list of technologies` horizontally. they are placed on the left side at the moment (please refer to the screenshot below) as it is in the original design in Figma. You can achieve this using the same trick we did with your  elements in the `.multi` section:

![Screenshot from 2022-02-18 02-58-25](https://user-images.githubusercontent.com/67608876/154578269-c4d1fcfb-d2b6-48f8-9068-63b7ec4befc2.png)